README.md

```diff
@@ -1,5 +1,102 @@
 # How to run
+Run ```docker run -p 5432:5432 -e POSTGRES_PASSWORD=password -e POSTGRES_USER=user postgres``` before starting
+the main Spring Boot application. You can start it using the run button on IntelliJ or in another terminal 
```

Comment:
"or in another terminal" - obvious. You're running all these commands in a terminal. Just have it like this:

```
mvn clean package
java -jar target/spring-boot-example-0.0.1-SNAPSHOT.jar server application.yml
```